Source/API/CBLJSON.m

```diff
@@ -125,7 +125,11 @@ + (size_t) estimateMemorySize: (id)object {
         // Thanks to DenNukem's answer in http://stackoverflow.com/questions/399527/
         sFormatter = [[NSDateFormatter alloc] init];
         sFormatter.dateFormat = @"yyyy-MM-dd'T'HH:mm:ss.SSSXXX";
+#ifdef __IPHONE_8_0
+        sFormatter.calendar = [[NSCalendar alloc] initWithCalendarIdentifier:NSCalendarIdentifierGregorian];
```

Comment:
Fixed. I saw that originally when I looked at the definition in the NSCalendar header, but then I looked around just to make sure and saw some reports of people having issues with it being unavailable on iOS. Looks like they were building with Xcode 5 against iOS 7 so it shouldn't be an issue anymore.